klog_test.go

```diff
@@ -473,7 +473,7 @@ func TestLogBacktraceAt(t *testing.T) {
 		// Need 2 appearances, one in the log header and one in the trace:
 		//   log_test.go:281: I0511 16:36:06.952398 02238 log_test.go:280] we want a stack trace here
 		//   ...
-		//   github.com/glog/glog_test.go:280 (0x41ba91)
+		//   github.com/kubernetes/klog/klog_test.go:280 (0x41ba91)
```

Comment:
`github.com/kubernetes/klog/klog_test.go` should be`k8s.io/klog/klog_test.go`. thanks!

Reply:
Is the correct line number?

Reply:
Line number does not matter, imo. But the package name matters.

@chrislovecnm Package name should be corrected. :)

Reply:
Others lgtm.

Reply:
@dims fixed